nanoFramework.DependencyInjection/DependencyInjection/ServiceProviderEngine.cs

```diff
@@ -160,7 +160,8 @@ private object Resolve(Type implementationType)
 
             if (constructorParameters == null)
             {
-                throw new InvalidOperationException();
+                throw new InvalidOperationException(
```

Comment:
One might think that I'm obsessed with this matter 🤪 but we really have to avoid messages in the exceptions. All the explanations should be in the Intellisense comments. 

I understand that on some of these cases could be helpful to pinpoint the type or a name. That's OK but lets rework those special cases with really telegraphic and succinct messages OK?

Reply:
I totally get it.  I have shortened the messages down to just enough words to provide context.  It reduced them by half but still provides direction.   Thanks again for considering these changes.